src/components/date-picker/calendar.tsx

```diff
@@ -42,10 +50,18 @@ export class Calendar extends React.Component<CalendarProps, {}> {
                         >
                             <i className="headerArrow headerArrowPrev" />
```

Comment:
when in month view, clicking on the arrows still changes the month and enough clicks will also change the year. I didn't see an easy way to change just the year so maybe on month view the arrows should change the year?